bin/src/daemon.rs

```diff
@@ -113,6 +122,34 @@ fn run_ctrl_c_thread(
     })
 }
 
+async fn update_ui(task_tracker: &TaskTracker,  config: &config::Config, mut ui_shutdown_rx: oneshot::Receiver<()>){
+    let image = match config.ui_level {
+        0 => {info!("silent UI!"); return},
+        1 => "subtle.png",
+        2 => "orca.gif",
+        _ => "orca.gif"
+    };
+
+    info!("spawning UI with {}", image);
+    task_tracker.spawn_blocking(move || {
+        let mut fb: Framebuffer = Framebuffer::new();
+        loop {
+            match ui_shutdown_rx.try_recv() {
+                Ok(_) => {
+                    info!("received UI shutdown");
+                    break;
+                },
+                Err(TryRecvError::Empty) => {},
+                Err(e) => panic!("what the fuck {e}")
```

Comment:
we _might_ want something more descriptive here lol

Reply:
done